src/MultiSelect/MultiSelect.svelte

```diff
@@ -22,11 +22,26 @@
   export let itemToString = (item) => item.text || item.id;
 
   /**
-   * Override the item name, title, labelText passed to the checkbox input
-   * @type {(item: MultiSelectItem) => { name?: string; labelText?: any; title?: string; }}
+   * Override the item name, title, labelText, or value passed to the user-selectable checkbox input as well as the hidden inputs.
+   * @type {(item: MultiSelectItem) => { name?: string; labelText?: any; title?:
+   * string; value?: string }}
    */
   export let itemToInput = (item) => {};
 
+  /**
+   * Set to `true` to only render selected options as hidden inputs for form submission.
+   * @type {boolean}
+   */
+  export let selectedOnly = false
```

Comment:
Could you run Prettier to format this file? I think that the trailing semicolons will be applied.